src/providers/whatsapp/dtos/Contacs.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { IsString, IsNotEmpty } from 'class-validator';
+
+export class Contacts {
+  @IsNotEmpty()
+  @IsString()
+  contactNumber: string;
+}
```

Comment:
disso também. Outro detalhe, não esquece que arquivo de DTO segue padrão camelcase, e a classe exportada tem que terminar com o "Dto":

```suggestion
import { IsString, IsNotEmpty } from 'class-validator';

export class ContactDTO {
  @IsNotEmpty()
  @IsString()
  contactNumber: string;
}
```

Reply:
ah blz